src/modules/form/components/viewable/DynamicViewField.tsx

```diff
@@ -114,6 +114,16 @@ const DynamicViewField: React.FC<DynamicViewFieldProps> = ({
       );
     case ItemType.Text:
     case ItemType.String:
+      if (item.component === Component.Ssn) {
+        return (
+          <TextContent
+            {...commonProps}
+            renderValue={(val) =>
+              `${val.slice(0, 3)}-${val.slice(3, 5)}-${val.slice(5, 9)}`
```

Comment:
I think we can skip this behavior, it would need more special casing to support partial SSN values